src/chip.tsx

```diff
@@ -144,7 +146,7 @@ const Chip: React.FC<ChipProps> = (props: ChipProps) => {
             )}
             paddingLeft={paddingLeft}
             paddingRight={paddingRight}
-            {...getPrefixedDataAttributes(dataAttributes, 'Chip')}
+            dataAttributes={dataAttributes}
```

Comment:
This wasn't working. Spreading data attributes over the Box resulted in ignored props

Reply:
hmmm... I think this is still wrong, because Box is applying `getPrefixedDataAttributes` again, so you'll end up with `data-data-xxx` attributes

Reply:
oh, right!